OCPBUGS-10695: Forbid IPv4-mapped IPv6 addresses in dual-stack (#57993)
openshift/openshift-docs: networking/ovn_kubernetes_network_provider/converting-to-dual-stack.adoc

A dual-stack network is supported on clusters provisioned on bare metal, IBM Power, {ibmzProductName} infrastructure, and single node OpenShift clusters.
====

[NOTE]
====

> When using dual-stack networking IPv4-mapped IPv6 addresses (in the format of e.g. `::FFFF:198.51.100.1`) must not be used in places where IPv6 address is required.

Reviewer: ```suggestion
While using dual-stack networking, you cannot use IPv4-mapped IPv6 addresses, such as `::FFFF:198.51.100.1`, where IPv6 is required.
```
